code/modules/client/preferences.dm

```diff
@@ -177,7 +177,15 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 			dat += "<a href='?_src_=prefs;preference=name;task=input'>[active_character.real_name]</a><BR>"
 
 			if(!(AGENDER in active_character.pref_species.species_traits))
-				dat += "<b>Gender:</b> <a href='?_src_=prefs;preference=gender'>[active_character.gender == MALE ? "Male" : "Female"]</a><BR>"
+				var/dispGender
+				if(active_character.gender == MALE)
+					dispGender = "Male"
+				else if(active_character.gender == FEMALE)
+					dispGender = "Female"
+				else
+					dispGender = "Other"
+				dat += "<b>Gender:</b> <a href='?_src_=prefs;preference=gender'>[dispGender]</a><BR>"
```

Comment:
Personally I think that the behaviour should be
- single button by default indicating gender, with ability to pick male, female or other.
- selecting other unlocks 2 additional buttons, 1 for pronoun and 1 for body type
- selecting male or female assigns the appropriate pronoun and bodytype

Reply:
Moved it so that it’s now only showing the body model preference if the player picks the "Other" gender. 

I think the other stuff would be out of reach for this PR although I can try to figure out a method to do so

Reply:
As long as the body model preference aligns to gender automatiaclly if you haven't selected other, then that will be fufilled anyway (In other words selecting female should be guaranteed to give you a female body, and same with male).

Reply:
Think it defaults to the current gender as body_model if it's not selected as "Other"